lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -305,18 +308,10 @@ def actual_permitted_parameter_names
           end
         end
 
-        def permit_called?
-          actual_permitted_parameter_names.any?
-        end
-
         def unpermitted_parameter_names
           expected_permitted_parameter_names - actual_permitted_parameter_names
         end
 
-        def verified_permitted_parameter_names
```

Comment:
I noticed this isn't used, but it seems like it would be useful to make sure that the actual list of permitted params matches the expected list, instead of looking for the actual being a superset of the expected.